api/database/connection.js

```diff
@@ -0,0 +1,52 @@
+const { Sequelize } = require("sequelize");
+
+require("dotenv").config();
+
+const createDbConnection = (
+  databaseName,
+  userName,
+  databasePassword,
+  dbHost,
+  dbDialect
+) => {
+  return new Sequelize(databaseName, userName, databasePassword, {
```

Comment:
i don't understand what u return, is it just the instance of Sequelize, is it a connection?
making a const var with indecative name will clear this up